packages/core/src/pipeline/config/triggers/ExecutionOptionsPageContent.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import { ExecutionOptionsPageContent } from './ExecutionOptionsPageContent';
+import type { IPipeline } from '../../../domain';
+
+describe('Execution Options Page Content', () => {
+  describe('Max Concurrent Options', () => {
+    let pipeline: IPipeline;
+    const setPipeline = (overrides: any = {}) => {
+      pipeline = {
+        application: 'test',
+        id: 'test1',
+        keepWaitingPipelines: false,
+        limitConcurrent: false,
+        maxConcurrentExecutions: 0,
+        name: 'test p 1', // @ts-ignore
+        parameterConfig: [], // @ts-ignore
+        stages: [], // @ts-ignore
+        triggers: [],
+        ...overrides,
+      };
+    };
+    const update = (changes: any = {}) => {
+      pipeline = {
+        ...pipeline,
+        ...changes,
+      };
+    };
+    describe('enabling max concurrent', () => {
+      it('sets keepWaitingPipelines to true if limitConcurrent and keepWaitingPipelines are both not truthy', () => {
+        setPipeline();
```

Comment:
I could do a 
```
beforeEach(() => {
  setPipeline()
});
```

and just recall it where I need overrides if anyone is adamant about it.

Reply:
Yeah, I think a `beforeEach` makes sense here.